definitions/npm/algoliasearch_v3.x.x/flow_v0.71.x-/algoliasearch_v3.x.x.js

```diff
@@ -0,0 +1,201 @@
+// @flow
+
+type JSONObject = mixed;
+type HTTPClient = any;
+type Callback<Response> = (error: ?Error, response: Response) => void;
+
+declare type $algolia$SearchIndexResponse = {|
+  hits: Array < any >,
+    page: number,
+      nbHits: number,
+        nbPages: number,
+          hitsPerPage: number,
+            processingTimeMS: number,
+              query: string,
+                parsed_query: string,
+                  params: string
+                    |};
```

Comment:
There is still quite a lot of weird formatting. Maybe worth running e.g trough prettier?